fe/fe-core/src/main/java/com/starrocks/common/proc/FrontendsProcNode.java

```diff
@@ -117,11 +118,16 @@ public static void getFrontendsInfo(Catalog catalog, List<List<String>> infos) {
                 info.add(Long.toString(fe.getReplayedJournalId()));
             }
             info.add(TimeUtils.longToTimeString(fe.getLastUpdateTime()));
-
             info.add(String.valueOf(isHelperNode(helperNodes, fe)));
-
             info.add(fe.getHeartbeatErrMsg());
 
+            if (fe.isAlive()) {
+                info.add(TimeUtils.longToTimeString(fe.getStartTime()));
+            } else {
+                info.add("NULL");
+            }
+            info.add(Version.STARROCKS_VERSION + "-" + Version.STARROCKS_COMMIT_HASH);
```

Comment:
This Version should return FE's version, not the SQL receiver's version.
So the version information should also get from heartbeat.

Reply:
done. I forgot this case before.